Python/Tic-Tac-Toe/Readme.md

```diff
@@ -0,0 +1,15 @@
+<h3>Concept Behind It :</h3>
```

Comment:
This is not what I told you. I told you to update the Python folder readme with your entry. Kindly delete this file

Reply:
ok